apps/studio/data/oauth/oauth-app-delete-mutation.ts

```diff
@@ -15,9 +14,12 @@ export async function deleteOAuthApp({ id, slug }: OAuthAppDeleteVariables) {
   if (!id) throw new Error('OAuth app ID is required')
   if (!slug) throw new Error('Organization slug is required')
 
-  const response = await delete_(`${API_URL}/organizations/${slug}/oauth/apps/${id}?type=published`)
-  if (response.error) throw response.error
-  return response
+  const { data, error } = await del('/platform/organizations/{slug}/oauth/apps/{id}', {
```

Comment:
We don't still need the `?type=published` here?

Reply:
ahh yeap - i checked API end but seems like they dont read the type query
https://github.com/supabase/infrastructure/blob/develop/api/src/routes/platform/organizations/slug/oauth/oauth-apps.controller.ts#L230